examples/dynamic_requirements.py

```diff
@@ -91,6 +92,20 @@ def run(self):
         with self.output().open('w') as f:
             f.write('Tada!')
 
+        # and in case data is rather long, consider wrapping the requirements
+        # in DynamicRequirements and optionally define a custom complete method
+        def custom_complete(complete_fn):
+            # example: Data() stores all outputs in the same directory, so avoid doing len(data) fs
+            # calls but rather check the only the first, and compare basenames for the rest
```

Comment:
perhaps typo? should this read `...but rather check only the first,...`? (too many `the`s)

Reply:
Fixed in 8c3e3a54.